ttnn/cpp/ttnn/deprecated/tt_dnn/op_library/moreh_nll_loss/moreh_nll_loss_op.cpp

```diff
@@ -317,10 +317,10 @@ Tensor moreh_nll_loss(
             reduction_mean,
             output_dtype,
             channel_size,
-            output_mem_config,
+            operation::DEFAULT_OUTPUT_MEMORY_CONFIG,
```

Comment:
Default config is DRAM. Is this intensional?

We generally recommend to use the optional memory config argument. And if it is nullopt, to take config from the first input tensor

Reply:
I didn't realize the default config was DRAM. I had set it as the default config to indicate that it would change to `nullopt` later, so I'll change it back to `output_mem_config`  for now.